src/traccuracy/metrics/_divisions.py

```diff
@@ -71,6 +69,10 @@ def __init__(self, frame_buffer=(0,)):
     def compute(self, data: Matched):
         """Runs `_evaluate_division_events` and calculates summary metrics for each frame buffer
 
+        Args:
+            matched_data (traccuracy.matchers.Matched): Matched object for set of GT and Pred data
+                Must meet the `needs_one_to_one` criteria
```

Comment:
TBH I still don't know what this means. We are saying that the division methods only work if the matching is one-to-one? I'd expect the metric to have the `needs_one_to_one` criteria and the matched to have... `one_to_one`? Saying that the Matched needs one-to-one doesn't make sense linguistically. But, we can relegate this to our validation issue #83

Reply:
Yeah I'm going to kick the can down the road for this one and leave it for #83. I agree that it's not clear, but I don't think there is an easy quick fix.